src/app/youtube/components/custom-item/custom-item.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, OnInit, Input, Output, EventEmitter, ChangeDetectionStrategy } from '@angular/core';
+import { ICustomItem } from 'src/app/shared/models/сustom-item.model';
+import { MAX_TITLE_LEN, PLACEHOLDER } from '../../../constants/common';
+
+@Component({
+  selector: 'app-custom-item',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  templateUrl: './custom-item.component.html',
+  styleUrls: ['./custom-item.component.scss']
+})
+export class CustomItemComponent implements OnInit {
```

Comment:
Don't you think that custom-item component and search item component are almost the same? Could inheritance be applied here?